pkg/providers/anthropic/chat_stream.go

```diff
@@ -2,15 +2,102 @@ package anthropic
 
 import (
 	"context"
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
 
 	"glide/pkg/api/schemas"
 	"glide/pkg/providers/clients"
 )
 
+type Client struct {
+	APIKey string
+}
+
 func (c *Client) SupportChatStream() bool {
-	return false
+	return true
+}
+
+func (c *Client) ChatStream(ctx context.Context, chatReq *schemas.ChatRequest) (clients.ChatStream, error) {
+	apiURL := "https://api.anthropic.com/v1/complete"
```

Comment:
Would this existing chatURL field work here? https://github.com/EinStack/glide/blob/develop/pkg/providers/anthropic/client.go

Seems like it's one and the same completion endpoint for both sync and streaming API in Anthropic just like in OpenAI